prow/cmd/gerrit/main.go

```diff
@@ -90,84 +90,114 @@ func gatherOptions(fs *flag.FlagSet, args ...string) options {
 }
 
 type syncTime struct {
-	val    time.Time
+	val    *client.LastSyncState
 	lock   sync.RWMutex
 	path   string
 	opener io.Opener
 	ctx    context.Context
 }
 
-func (st *syncTime) init() error {
-	fmt.Println(st.val)
+func (st *syncTime) init(hostProjects client.ProjectsFlag) error {
+	logrus.Debug(st.val)
```

Comment:
Can we make this a little better, like by using fields to log what component it is assicated with, etc (as we generally do)?

Reply:
done